.gitignore

```diff
@@ -1,4 +1,42 @@
 
 .DS_Store
 
-/db/apps/*/build/*
\ No newline at end of file
+# Build artifacts
```

Comment:
I would also delete and ignore the .xar collection no need to carry this baggage `xars/resources/js/attestations.js` is a heavy scratcher